src/api/json/catalog.json

```diff
@@ -1570,10 +1570,28 @@
       "url": "https://json.schemastore.org/first-timers.json"
     },
     {
-      "name": "Foundry VTT - Manifest",
-      "description": "JSON schema for Foundry VTT system.json and module.json files.",
-      "fileMatch": ["system.json", "module.json"],
-      "url": "https://json.schemastore.org/foundryvtt-manifest.json"
+      "name": "Foundry VTT - Base package Manifest",
+      "description": "Base schema for Module/system/World manifest schemas to inherit from.",
+      "fileMatch": [],
+      "url": "https://json.schemastore.org/foundryvtt-base-package-manifest.json"
+    },
+    {
+      "name": "Foundry VTT - Module Manifest",
+      "description": "JSON schema for Foundry VTT module.json files.",
+      "fileMatch": ["module.json"],
+      "url": "https://json.schemastore.org/foundryvtt-module-manifest.json"
+    },
+    {
+      "name": "Foundry VTT - System Manifest",
+      "description": "JSON schema for Foundry VTT system.json files.",
+      "fileMatch": ["system.json"],
```

Comment:
This file name seems generic. Aren't there any conventions of folder structure you can add to the globbing pattern?

Reply:
Sadly no, there isn't any.  The general file structure of Foundry packages is just `someDirectory/projectName/potentialInnerDirectory/system.json`.

Examples:

https://github.com/moo-man/WrathAndGlory-FoundryVTT - in root folder of package, `/`

https://gitlab.com/foundryvtt_pathfinder1e/foundryvtt-pathfinder1 - in `/public/`

https://gitlab.com/peginc/swade/ - in `/src/`

Reply:
It's too bad it's this generic. We can't add the `fileMatch` to the catalog due to the false positive matches.